src/extension.ts

```diff
@@ -56,6 +56,15 @@ export async function activate(context: ExtensionContext): Promise<RExtension> {
         rWorkspace
     );
 
+    // creates a custom context value for the workspace view
+    // only shows view when session watcher is enabled
+    const sessionBool = workspace.getConfiguration('r')['sessionWatcher'];
+    if (sessionBool === true) {
+        void commands.executeCommand('setContext', 'r.WorkspaceViewer:show', true);
```

Comment:
Here can reduce if statement by 
```ts
void commands.executeCommand('setContext', 'r.WorkspaceViewer:show', sessionBool);
```
And I suggest the name `enabledSessionWatcher`instead of `sessionBool`

Reply:
Much more elegant, thank you!

Reply:
Good, thank you for the early fix